scripts/setup-macos.sh

```diff
@@ -77,6 +77,9 @@ function install_build_prerequisites {
     python3 -m venv ${PYTHON_VENV}
   fi
   source ${PYTHON_VENV}/bin/activate; pip3 install cmake-format regex pyyaml
+  wget -O ccache.tar.gz https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2-darwin.tar.gz
+  tar -xf ccache.tar.gz
+  mv ccache-4.10.2-darwin/ccache /usr/local/bin/
```

Comment:
Mmhh, this might be restricted. For the same reason we want INSTALL_PREFIX on macOS.

Reply:
Should be okay. We install minio here as well.